### PR TITLE
🐛 amp-access: Keeps trying to close window for a minute

### PR DESCRIPTION
**Pull request by ChrisAntaki**
This PR updates `amp-access`'s login done dialog to keep trying to close its window for a minute.

Sometimes `window.close()` is ignored by the browser for a stretch of time. For instance, iOS ignores the method when there is a "Save Password" or "Update Password" prompt open. (https://github.com/ampproject/amphtml/issues/11369)

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
Oops, I didn't mean to remove you as a reviewer, John

**Comment by ChrisAntaki** [comment]
I just verified this works locally with a real iPhone and also a real Android device

File: extensions/amp-access/0.1/amp-login-done-dialog.js [hunk 1]
```diff
@@ -203,9 +203,23 @@ export class LoginDoneDialog {
...
+        // Ignore.
+      }
+    }, 100);
+
     // Give the opener a chance to close the dialog, if not, show the
     // close button.
     this.win.setTimeout(() => {
+      clearInterval(windowCloseInterval);
       this.postbackError_(new Error('Failed to close the dialog'));
     }, 3000);
```
Comment (jpettitt) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Is three seconds long enough to catch most users handling the password save popup? Would there be any harm in making it longer (5? 10?). @dvoytenko WDYT is a good value here?

Comment (ChrisAntaki) [reply extensions/amp-access/0.1/amp-login-done-dialog.js]: That's a good point. Honestly, I think it'd be safe to remove this `clearInterval` and have it keep trying indefinitely

Comment (jpettitt) [reply extensions/amp-access/0.1/amp-login-done-dialog.js]: Let's go for 60 seconds.  i don't like loops that run forever. :-)

File: extensions/amp-access/0.1/amp-login-done-dialog.js [hunk 2]
```diff
@@ -203,9 +203,23 @@ export class LoginDoneDialog {
...
+    // For instance, iOS ignores the method when there is a
+    // "Save Password" or "Update Password" prompt open.
+    // https://github.com/ampproject/amphtml/issues/11369
+    const windowCloseInterval = this.win.setInterval(() => {
+      try {
+        this.win.close();
+      } catch (e) {
+        // Ignore.
+      }
+    }, 100);
```
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Let's definitely increase interval. Maybe 500.

Comment (ChrisAntaki) [reply extensions/amp-access/0.1/amp-login-done-dialog.js]: Done!

File: extensions/amp-access/0.1/amp-login-done-dialog.js (hunk 1 above)
Comment (ChrisAntaki) [reply extensions/amp-access/0.1/amp-login-done-dialog.js]: Done!

**Pull request merged**